dev/README.md

```diff
@@ -0,0 +1,22 @@
+# CompreFace
+
+## Local start all in prod mode
+
+Run script to start docker container:
+
+- `./scripts/start.sh` for unix OS
```

Comment:
As I know this scripts are needed only for UI developers.
Backend team just start it with docker-compose up --build - see https://github.com/exadel-inc/CompreFace/tree/develop#getting-started-for-contributors - probably we just need to move it here for backend develoeprs

Reply:
Could you please clarify what are you mean by talking about `need to move it here `?